ignite/metrics/confusion_matrix.py

```diff
@@ -323,3 +323,31 @@ def ignore_index_fn(dice_vector: torch.Tensor) -> torch.Tensor:
         return MetricsLambda(ignore_index_fn, dice)
     else:
         return dice
+
+
+def JaccardIndex(cm: ConfusionMatrix, ignore_index: Optional[int] = None) -> MetricsLambda:
+    r"""Calculates the Jaccard Index using :class:`~ignite.metrics.ConfusionMatrix` metric.
+    This is the same like the IoU metric
+    .. math:: \text{J}(A, B) = \frac{ \lvert A \cap B \rvert }{ \lvert A \cup B \rvert }
```

Comment:
This is badly rendered: https://deploy-preview-1682--pytorch-ignite-preview.netlify.app/metrics.html#ignite.metrics.JaccardIndex

Reply:
<img width="812" alt="Screen Shot 2021-02-22 at 11 26 19 PM" src="https://user-images.githubusercontent.com/2458523/108783464-73360980-7565-11eb-96c2-7a00a9512e5c.png">

Seems to be fixed now!